### ISSUE TITLE
Multiple list.data tests break on Windows

### ISSUE THREAD

**Issue by connectedblue** [KentonWhite/ProjectTemplate#197]
Hi @Hugovdberg 
There are a lot of tests that fail when they are executed on a windows machine.  I'm struggling to test a new feature before submission.

I'm not sure exactly what the tests are doing, but the failure report below looks like there is some kind of file path separator issue with an extra `\\` that appears and causes the tests to fail.

Is it possible to fix the tests so they run cross platform?

```
1. Failure: available data is listed correctly with default configuration (@test-list.R#84) ---
data.files$filename not equal to test.df$filename.
1/3 mismatches
x[1]: "file1b64623d2542.csv"
y[1]: "\\file1b64623d2542.csv"


2. Failure: available data is listed correctly with default configuration (@test-list.R#85) ---
data.files$varname not equal to test.df$varname.
1/3 mismatches
x[1]: "file1b64623d2542"
y[1]: "\\file1b64623d2542"


3. Failure: available data is listed correctly with recursive_loading = TRUE (@test-list.R#146) 
data.files$filename not equal to test.df$filename.
2/3 mismatches
x[1]: "file1b6467de1cc3.csv"
y[1]: "\\file1b6467de1cc3.csv"

x[3]: "test/file1b646769158e.csv"
y[3]: "test\\file1b646769158e.csv"


4. Failure: available data is listed correctly with recursive_loading = TRUE (@test-list.R#147) 
data.files$varname not equal to test.df$varname.
2/3 mismatches
x[1]: "file1b6467de1cc3"
y[1]: "\\file1b6467de1cc3"

x[3]: "test.file1b646769158e"
y[3]: "test\\file1b646769158e"


5. Failure: available data is listed correctly with data_ignore (@test-list.R#198) ------------
data.files$filename not equal to test.df$filename.
1/4 mismatches
x[1]: "file1b644ef33821.csv"
y[1]: "\\file1b644ef33821.csv"


6. Failure: available data is listed correctly with data_ignore (@test-list.R#228) ------------
data.files$filename not equal to test.df$filename.
2/4 mismatches
x[1]: "file1b644ef33821.csv"
y[1]: "\\file1b644ef33821.csv"

x[3]: "test/file1b64161b5021.csv"
y[3]: "test\\file1b64161b5021.csv"


7. Failure: available data is listed correctly with data_ignore (@test-list.R#254) ------------
data.files$filename not equal to test.df$filename.
1/4 mismatches
x[1]: "file1b644ef33821.csv"
y[1]: "\\file1b644ef33821.csv"


8. Failure: available data is listed correctly with data_ignore (@test-list.R#283) ------------
data.files$filename not equal to test.df$filename.
2/4 mismatches
x[1]: "file1b644ef33821.csv"
y[1]: "\\file1b644ef33821.csv"

x[3]: "test/file1b64161b5021.csv"
y[3]: "test\\file1b64161b5021.csv"


9. Failure: available data is listed correctly with data_ignore (@test-list.R#314) ------------
data.files$filename not equal to test.df$filename.
2/4 mismatches
x[1]: "file1b644ef33821.csv"
y[1]: "\\file1b644ef33821.csv"

x[3]: "test/file1b64161b5021.csv"
y[3]: "test\\file1b64161b5021.csv"


10. Failure: available data is listed correctly with data_ignore (@test-list.R#345) -----------
data.files$filename not equal to test.df$filename.
2/4 mismatches
x[1]: "file1b644ef33821.csv"
y[1]: "\\file1b644ef33821.csv"

x[3]: "test/file1b64161b5021.csv"
y[3]: "test\\file1b64161b5021.csv"


11. Failure: available data is listed correctly with data_ignore (@test-list.R#377) -----------
data.files$filename not equal to test.df$filename.
2/4 mismatches
x[1]: "file1b644ef33821.csv"
y[1]: "\\file1b644ef33821.csv"

x[3]: "test/file1b64161b5021.csv"
y[3]: "test\\file1b64161b5021.csv"


12. Failure: cached data is listed correctly as already cached (@test-list.R#427) -------------
data.files$filename not equal to test.df$filename.
1/3 mismatches
x[1]: "file1b643a596404.csv"
y[1]: "\\file1b643a596404.csv"


13. Failure: cached data is listed correctly as already cached (@test-list.R#453) -------------
data.files$filename not equal to test.df$filename.
1/3 mismatches
x[1]: "file1b643a596404.csv"
y[1]: "\\file1b643a596404.csv"


14. Failure: cached data created during munging listed as cached only (@test-list.R#509) ------
data.files$filename not equal to test.df$filename.
1/4 mismatches
x[1]: "file1b6429620b9.csv"
y[1]: "\\file1b6429620b9.csv"


15. Failure: cached data created during munging listed as cached only (@test-list.R#510) ------
data.files$varname not equal to test.df$varname.
1/4 mismatches
x[1]: "file1b6429620b9"
y[1]: "\\file1b6429620b9"
```

**Comment by Hugovdberg** [comment]
interesting, both `filename` and `varname` are affected. I will check to see what's going on internally.

**Comment by connectedblue** [comment]
thanks .. let me know if you have a branch you want me to test against.

**Comment by Hugovdberg** [comment]
This should be fixed with the merge of #199, closing this issue.
